js/markdown/shared/MarkdownCode.svelte

```diff
@@ -188,4 +188,8 @@
 	span.md :global(.md-header-anchor > svg) {
 		color: var(--body-text-color-subdued);
 	}
+
+	span :global(table) {
```

Comment:
My understanding is that this means that the the rule applies to <table> elements globally, not just within the Markdown component. Is that what we want?